### PR TITLE
Remove documents filepath column from db

### PR DESCRIPTION
**Pull request by lowellrex**
Connects #778.

Confirmed that we are using the new strategy to generate document file paths that we incorporated in #736 and released on Monday by hopping into the rails console on a production box and running
```
Document.find_by_sql("select * from documents where filepath is not null order by id desc limit 1")[0].created_at
```
revealing that the most recently created document row that has a filepath value was created at ~10am Monday which lines up with when the initial change (#774) was pushed live.

### CODE REVIEW

File: app/models/download.rb [hunk 1]
```diff
@@ -164,7 +164,7 @@ def package_filename
   def reset!
     Download.transaction do
       update_attributes!(status: :pending_documents)
-      documents.update_all(filepath: nil, download_status: 0, completed_at: nil)
+      documents.update_all(download_status: 0, completed_at: nil)
```
Comment (aroltsch) [inline app/models/download.rb]: Ugh, can we have a situation where a migration will finish but the application will still use the "old" code? Maybe we need to deploy this change first and then the migration.

Comment (lowellrex) [reply app/models/download.rb]: Hmmm I would guess this is unlikely but it definitely reveals that I don't know exactly what the build process looks like. I don't imagine we would start the rails server until we have run all the migrations, but that is a total guess. Where would the build code live? [Appeals deployment repo](https://github.com/department-of-veterans-affairs/appeals-deployment)?

Comment (aroltsch) [reply app/models/download.rb]: https://wiki.openstack.org/wiki/DBMigrationBestPractices#Best_Practices

Comment (aroltsch) [reply app/models/download.rb]: we should separate this change from the migration to ensure backwards compatibility.

**Comment by lowellrex** [comment]
This PR should not be deployed until [the related PR](https://github.com/department-of-veterans-affairs/caseflow-efolder/pull/800) that removes all references to `Document.filepath` from the app logic is running live.

**Pull request merged**